### DISCUSSION TITLE
requestFullscreen not available

### DISCUSSION THREAD

**Discussion by ambitiouschris** [videojs/video.js#7575] [Q&A]
I am pretty new the video js package.  We are getting an error when trying to click on the full screen button on an iphone.
We are using version 7.8.4 of the videojs

TypeError: e[l[a.requestFullscreen]] is not a function. (In 'e[l[a.requestFullscreen]]()', 'e[l[a.requestFullscreen]]' is undefined)
  at apply (../fscreen.js:53:52)
  at Qt ([native code])
  at $t ([native code])
  at r (../helpers.ts:87:17)
...
(14 additional frame(s) were not displayed)

It looks like the error is happening in fscreen.

  requestFullscreen: element => element[vendor[key.requestFullscreen]](),

**Comment by gkatsev** [comment]
Seems to be working for me locally. Can you provide a live test page?
Also, are you able to upgrade to a newer version of Video.js? 7.8.4 is a year and a half old at this point.
